### DISCUSSION TITLE
Can't install git in homebrew because it requires git

### DISCUSSION THREAD

**Discussion by XxMicrowavexX** [Homebrew/discussions#3228] [Getting started]
So I install brew and wanna run brew install git and this happens
![Screen Shot 2022-04-30 at 11 46 40 AM](https://user-images.githubusercontent.com/76092869/166081601-59d853d8-53d6-4ee0-afb0-31e178a773d2.png)

**Comment by carlocab** [comment]
What is the output of `brew config` and `brew doctor`?
